.yamllint.yml

```diff
@@ -21,8 +21,6 @@ rules:
 
       /tests/tests_ethtool_ring_initscripts.yml
 
-      /tests/tests_route_device_initscripts.yml
```

Comment:
And why this line was dropped?

Reply:
Right - all .ansible-lint and .yamllint.yml changes should be made https://github.com/linux-system-roles/.github/blob/main/inventory/host_vars/network.yml

Reply:
I fixed this, extra_vars: in ansible lint have a different order but that's expected

Reply:
I see.